### PR TITLE
🎈 10.6.0

### PR DESCRIPTION
**Pull request by chapulina**
# 🎈 Release

Preparation for 10.6.0 release.

Comparison to 10.5.0: https://github.com/osrf/sdformat/compare/sdformat10_10.5.0...sdf10

Needed by https://github.com/ignitionrobotics/ign-sensors/pull/153

## Checklist
- [x] Asked team if this is a good time for a release
- [x] There are no changes to be ported from the previous major version
- [x] No PRs targeted at this major version are close to getting in
- [x] Bumped minor for new features, patch for bug fixes
- [x] Updated changelog
- [ ] ~~Updated migration guide (as needed)~~
- [ ] ~~Link to PR updating dependency versions in appropriate repository in [ignition-release](https://github.com/ignition-release) (as needed): <LINK>~~

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,85 @@
...
+    * [Pull request #681](https://github.com/ignitionrobotics/sdformat/pull/681)
+
+1. Upgrade cpplint and fix new errors
+    * [Pull request #680](https://github.com/ignitionrobotics/sdformat/pull/680)
+
+1. BUG: add missing plugin element to include
+    * [Pull request #675](https://github.com/ignitionrobotics/sdformat/pull/675)
+
+1. Update codecov badge url
+    * [Pull request #678](https://github.com/ignitionrobotics/sdformat/pull/678)
```
Comment (scpeters) [inline Changelog.md:24]: we redacted this from the 9.6.1 changelog (see https://github.com/ignitionrobotics/sdformat/commit/32ce16a81e707a5000a13cbec686371689b336f7 and https://github.com/ignitionrobotics/sdformat/pull/691#discussion_r703759778)

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,85 @@
...
+
+1. Added comment reminder to update functions
+    * [Pull request #677](https://github.com/ignitionrobotics/sdformat/pull/677)
+
+1. Port to new codecov configuration
+    * [Pull request #676](https://github.com/ignitionrobotics/sdformat/pull/676)
+
+1. Add pull request triage to sdf9
+    * [Pull request #673](https://github.com/ignitionrobotics/sdformat/pull/673)
```
Comment (scpeters) [inline Changelog.md:33]: we redacted these two as well (#676 and #673)

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,85 @@
...
+
+1. Adds enable_metrics flag to Sensor.
+    * [Pull request #665](https://github.com/ignitionrobotics/sdformat/pull/665)
+
+1. Add GPS sensor to sdf9
```
Comment (scpeters) [inline Changelog.md:38]: the commit message will always say GPS, but should we change `GPS` to `NavSat` to match the renaming applied in the code?

**Comment by chapulina** [comment]
@scpeters , addressed your feedback on 2fd4065

**Review by scpeters — APPROVED** [review]

**Pull request merged**